android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/StatsIgUtil.kt

```diff
@@ -13,6 +13,13 @@ object StatsIgUtil {
 
     private const val FEATURE_GATE_DELETE_TODO_ACCESS = "enable_delete_todo"
     private const val TEST_MOBILE_DYNAMIC_CONFIG = "mobile_dynamic_config"
+    const val LOG_EVENT_APP_OPENED = "app opened"
```

Comment:
I'd name these `APP_OPENED`, `TODO_CREATED`, etc.  The `LOG_EVENT` piece is redundant

Reply:
I will make the change